projects/kubernetes/autoscaler.config.json

```diff
@@ -0,0 +1,49 @@
+{
+  "tag_specification": {
+    "cluster": "https://api.ci.openshift.org",
+    "namespace": "openshift",
+    "name": "origin-v3.10",
+    "tag": "",
+    "tag_overrides": {}
+  },
+  "base_rpm_images": {
+    "base": {
+      "cluster": "https://api.ci.openshift.org",
+      "namespace": "openshift",
+      "name": "origin-v3.10",
+      "tag": "base"
+    }
+  },
+  "test_base_image": {
+    "namespace": "ci",
+    "name": "release-with-clonerefs",
+    "tag": "golang-1.9"
+  },
+
+  "binary_build_commands": "mkdir -p $GOPATH/src/k8s.io; ln -s $(pwd) $GOPATH/src/k8s.io/autoscaler; make build",
+  "test_binary_build_commands": "mkdir -p $GOPATH/src/k8s.io; ln -s $(pwd) $GOPATH/src/k8s.io/autoscaler; OS_GOFLAGS='-race' make build",
+  "rpm_build_commands": "make build-rpms",
+
+  "images": [
+    {
+      "from": "base",
+      "to": "cluster-autoscaler",
+      "context_dir": "images/cluster-autoscaler/"
+    }
+  ],
+
+  "tests": [
+    {
+      "as": "unit",
+      "from": "test-bin",
+      "commands": "cd $GOPATH/src/k8s.io/autoscaler/cluster-autoscaler; go test ./..."
```

Comment:
I swear we were setting the working dir in the image to point to where we cloned the code...

Reply:
Or is this because of `k8s.io`

Reply:
@stevekuznetsov because `k8s.io`